Dockerfile

```diff
@@ -1,12 +1,12 @@
-FROM ubuntu
+FROM python:3.9.0-buster
 
-COPY requirements.txt requirements.txt
+RUN apt-get update
 
-RUN apt update && apt install -y git make unzip curl g++ python3 pip wget
-RUN pip3 install -r requirements.txt
+RUN pip3 install Pillow
```

Comment:
Isn't it better to use "requirements.txt" so that we won't touch a Docker file but just edit "requirements.txt"?
All the python packaging isse would be stored in "reqiurments.txt".